src/main/java/website/frontrow/util/CollisionHandler.java

```diff
@@ -14,8 +11,9 @@
 import website.frontrow.logger.Logable;
 
 /**
- * This Class is for handling collisions during the game.
+ * Computes collisions during the game.
  */
+// TODO: Refactor name to CollisionComputer
```

Comment:
Please finish this todo. It's a bit confusion to have a CollisionHandler and a RealCollisionHandler.